ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -73,7 +73,7 @@ public async Task<IActionResult> OnPostAsync()
                 return NotFound();
             }
 
-            if (string.IsNullOrEmpty(Team.PrimaryContactEmail))
+            if (string.IsNullOrWhiteSpace(Team.PrimaryContactEmail))
```

Comment:
I think Required attribute would take care of these automatically (but check me on it)

Reply:
Okay, I'll take out this and the empty check after it and put required attributes on them both instead since, per Kenny's recommendation, I'll be doing all the length limits in the database instead.  Look for that in my next/final commit.